src/api/secrets.go

```diff
@@ -546,7 +546,9 @@ func IsValidSecretType(secretType string) error {
 func checkClustersBeforeDelete(orgId uint, secretId string) error {
 	// TODO: move these to a struct and create them only once upon application init
 	secretValidator := providers.NewSecretValidator(secret.Store)
-	clusterManager := cluster.NewManager(clusteradapter.NewClusters(global.DB()), secretValidator, cluster.NewNopClusterEvents(), nil, nil, nil, log, errorHandler)
+	klusters := clusteradapter.NewClusters(global.DB())
```

Comment:
??

Reply:
`clusters` was already in use and naming is hard

Reply:
How about not putting it in a variable then?

Reply:
Because the value is used in more than one place?

Reply:
what about clusteradapter.Store interface extend the clusteradapter.Clusters interface? Or adding a method Clusters() to it which exposes the internal `clusters` field.